server/pbench/bin/index-pbench

```diff
@@ -598,16 +626,29 @@ class ToolData(object):
         column coordinate."""
         pass
 
+    def _make_source_json(self):
+        for df in self.json_files:
+            payload = json.loads(open(os.path.join(_tmpdir, df['path'])).read())
+            for item in payload:
+                item['@metadata'] = self.run_metadata
+                ts = datetime.utcfromtimestamp(item['@timestamp'])
+                ts_str = ts.strftime("%Y-%m-%dT%H:%M:%S.%f%z")
+                item['@timestamp'] = ts_str
```

Comment:
If we need to transform the data, this is where that's to be done. Also, the timestamp in prometheus data is in the correct form for this transformation (seconds since epoch), but if the format is different in other data
then this will barf. We need to check it.